### DISCUSSION TITLE
如何嵌入至普通win32项目？

### DISCUSSION THREAD

**Discussion by KnIfER** [lc-soft/LCUI#296] [Q&A]
需要静态lib、头文件，如有可能的话，希望能生成vcproject，然后将lcui.vcxproj拖入我的解决方案.sln，不要用npm一下生成那么多文件，看不来。

试了下demo，缩放一下窗口cpu占用近百了。输入框没有文本选择，这是没有开启还是没实现？

**Comment by lc-soft** [comment]
发行版页面内有提供头文件和 dll 文件下载链接：https://github.com/lc-soft/LCUI/releases

目前未提供静态 lib，当时这么做的考虑是链接其它依赖库比较麻烦，编译成 dll 就不用管其它依赖库了。

demo CPU 占用率的问题，主要原因是图形绘制都是纯 CPU 运算的，无硬件加速。

至于文本框的文本选择功能，是没实现。
